docs/style.less

```diff
@@ -195,17 +195,19 @@ header {
             font-size: 2em;
             margin: 0;
             font-weight: normal;
+            white-space: normal;
 
             span.hashtag_style {
                 font-weight: bold;
                 padding: 0px 20px;
                 border-radius: 20px;
+                white-space: nowrap;
             }
         }
     }
 
     nav {
-        margin: 0 0 1rem 0;
+        margin: 1rem 0 1rem 0;
```

Comment:
Not keen on margin-top 1rem here. I think might be too much space on desktop and KK etc asked us to reduce that space this morning. was inc. in one of my earlier commits.

Reply:
@david-raine is this needed / wanted for other devices? If so could we put it in a media query and have 0 for desktop possibly?

Reply:
going to merge this as we want the good changes in ! Thanks david